ReactiveCocoa/Swift/Property.swift

```diff
@@ -60,6 +60,17 @@ public struct AnyProperty<Value>: PropertyType {
 	}
 }
 
+/// Maps the current value and all subsequent values to a new value.
+extension PropertyType {
+	public func map<U>(transform: Value -> U) -> AnyProperty<U> {
+		let signalProducer = SignalProducer<Value, NoError> { observer, disposable in
+			disposable += ActionDisposable { self }
+			disposable += self.producer.start(observer)
+		}
+		return AnyProperty(initialValue: transform(value), producer: signalProducer.map(transform))
```

Comment:
`transform` is applied twice to the same value, because `property.producer` sends its current value on subscription. Seems like it can be solved by observing `signal` instead of `producer` on line 68.